cmake/scripts/tvos/ArchSetup.cmake

```diff
@@ -0,0 +1,57 @@
+if(NOT CMAKE_TOOLCHAIN_FILE)
+  message(FATAL_ERROR "CMAKE_TOOLCHAIN_FILE required for tvos. See ${CMAKE_SOURCE_DIR}/cmake/README.md")
+endif()
+
+set(CORE_MAIN_SOURCE ${CMAKE_SOURCE_DIR}/xbmc/platform/darwin/tvos/MainApplication.mm)
+
+set(ARCH_DEFINES -D_LINUX -DTARGET_POSIX -DTARGET_DARWIN -DTARGET_DARWIN_TVOS)
```

Comment:
When this was bundled in the other PR, someone commented this line is actually duplicated from the platform/tvos/defines.txt
Therefore, should just remove it

Reply:
Ok to remove the line 7, but this duplicate also happen in the ArchSetup.cmake iOS file ...
So we remove this line in tvOS but we keep the duplicate line in iOS?

Reply:
iOS cleanup for another PR